examples/spring-boot-minimal-kotlin/build.gradle

```diff
@@ -0,0 +1,47 @@
+buildscript {
+    repositories {
+        gradlePluginPortal()
+    }
+    dependencies {
+        classpath "org.jetbrains.kotlin:kotlin-allopen:${managedVersions['org.jetbrains.kotlin:kotlin-allopen']}"
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:${managedVersions['org.jetbrains.kotlin:kotlin-gradle-plugin']}"
+        classpath "org.springframework.boot:spring-boot-gradle-plugin:${managedVersions['org.springframework.boot:spring-boot-gradle-plugin']}"
+    }
+}
+
+apply plugin: "org.jetbrains.kotlin.jvm"
```

Comment:
Let's go ahead and use `plugins` block instead of old style.

Reply:
I'm trying to solve it. 🤔

Reply:
@anuraaga I found it difficult to remove old style. 
It seems possible that [gradle/gradle#9830](https://github.com/gradle/gradle/issues/9830) should be resolved.